projects/fmcadc5/Readme.md

```diff
@@ -0,0 +1,8 @@
+# FMCADC5 HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/ad-fmcadc5-ebz)
+  * Parts : [12-Bit, 2.6 GSPS/2.5 GSPS/2.0 GSPS, 1.3 V/2.5 V Analog-to-Digital Converter](https://www.analog.com/ad9625)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcadc5-ebz
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcadc5-ebz/quickstart/microblaze
```

Comment:
The HDL description for the project is also at https://wiki.analog.com/resources/eval/user-guides/ad-fmcadc5-ebz